### PR TITLE
Add skeletons needed for hibernate orm rest extension to generate CRU…

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

File: qshift/templates/quarkus-extensions/skeletons/src/main/java/MyEntity.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{% if values.enable_rest %}
+package org.acme;
```
Comment (cmoulliard) [inline qshift/templates/quarkus-extensions/skeletons/src/main/java/MyEntity.java:2]: Why don't you set the package name according to the value `java_package_name` passed using the parameter ?

File: qshift/templates/quarkus-extensions/skeletons/src/main/java/MyEntityResource.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{% if values.enable_rest %}
+package org.acme;
```
Comment (cmoulliard) [inline qshift/templates/quarkus-extensions/skeletons/src/main/java/MyEntityResource.java:2]: Why don't you set the package name according to the value `java_package_name` passed using the parameter ?

**Review by cmoulliard — CHANGES_REQUESTED** [review]
The file name used as package_folder_name should be set using `${{values.java_package_name}}`

File: qshift/templates/quarkus-extensions/skeletons/src/main/java/MyEntityResource.java (hunk 2 above)
Comment (aureamunoz) [reply qshift/templates/quarkus-extensions/skeletons/src/main/java/MyEntityResource.java]: done

File: qshift/templates/quarkus-extensions/skeletons/src/main/java/MyEntity.java (hunk 1 above)
Comment (aureamunoz) [reply qshift/templates/quarkus-extensions/skeletons/src/main/java/MyEntity.java]: done

**Review by cmoulliard — APPROVED** [review]

**Pull request merged**